releases/latest/full/Dockerfile.ubi.adoptopenjdk15

```diff
@@ -81,8 +82,7 @@ RUN if [ "$OPENJ9_SCC" = "true" ]; then populate_scc.sh; fi \
     && chmod -R g+rwx /opt/ol/wlp/output
 
 #These settings are needed so that we can run as a different user than 1001 after server warmup
-ENV RANDFILE=/tmp/.rnd \
-    OPENJ9_JAVA_OPTIONS="-Xshareclasses:name=liberty,nonfatal,cacheDir=/output/.classCache/"
+ENV RANDFILE=/tmp/.rnd
```

Comment:
@ymanton Should JDK 15 images get this update? JDK 14 images didn't (i.e. in 20.0.0.9), so just wanted to make sure.

Reply:
Yes they should. The JDK 14 images have broken SCC permissions so we want to continue to build and use a Liberty SCC in that case. The JDK 15 images have a working SCC so we can use it.